src/compilers/template_engine.ts

```diff
@@ -2,12 +2,32 @@ const decoder = new TextDecoder();
 
 export default class TemplateEngine {
 
+  /**
+   * @description
+   *     A property to hold the base path to the template.
+   *
+   * @property string views_path
+   */
   public views_path: string = "";
 
+  // FILE MARKER: CONSTRUCTOR //////////////////////////////////////////////////
+
+  /**
+   * @description
+   *     Construct an object of this class.
+   *
+   * @param number code
+   *     The HTTP response code associated with this exception.
+   * @param string message
+   *     (optional) The exception message.
+   */
```

Comment:
Little picky.. but doesn't seem to be a `code` param passed in?

Reply:
LOL copy paste.

Reply:
fixed